Dockerfile

```diff
@@ -5,8 +5,8 @@ WORKDIR /usr/src/app
 RUN apt-get update && \
     apt-get -y install build-essential git curl python3-pip && \
     apt-get clean && \
-    pip3 install awscli
+    pip3 install awscli \
+    bundle install
```

Comment:
There's nothing to `bundle install` until after you've cloned the `zendesk-scripts` repo and are in that directory (line 12).

Reply:
Thanks Issy, I'll move it.